spec/descriptions/createGroup.md

```diff
@@ -0,0 +1,20 @@
+Creates a group on the tenant unit. Each group entry also needs a `Permission Set`.
```

Comment:
I suggest: `Creates a group on the tenant. Each group entry also needs a `Permission Set` per unit.` 
to avoid misunderstanding as if there could be groups per unit.